include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -1094,6 +1094,12 @@ struct OrtApi {
   ORT_API2_STATUS(OrtSessionOptionsAppendExecutionProvider_CUDA,
                   _In_ OrtSessionOptions* options, _In_ OrtCUDAProviderOptions* cuda_options);
 #endif  // USE_CUDA
+
+  /**
+   * Use this API to configure the global thread pool options to be used in the call to CreateEnvWithGlobalThreadPools.
+   * Set denormal as zero.
```

Comment:
Would be good to document: 1. when would someone want to call this API, what alternatives exist and what's the impact of using this (like loss of accuracy)? 2. this applies to both inter and intra op threadpools.

Reply:
Add details here.